### PR TITLE
Update export-user-data-project-server-subscription-edition.md

### PR DESCRIPTION
**Pull request by NadinMerali**
Adding documentation about an unsupported scenario

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Project/export-user-data-project-server-subscription-edition.md [hunk 1]
```diff
@@ -585,6 +585,7 @@ After you run the `ExportProjectUserContent` PowerShell script successfully, you
 
     > [!Note]
     > You can open the .mpp file with Project Professional 2021, Project Professional 2019, or the Project Online Desktop client.
+    > Saving the exported .mpp files back to Project Online or Project server is not supported.
```
Comment (JohanFreelancer9) [inline Project/export-user-data-project-server-subscription-edition.md:588]:
```suggestion
    > Saving the exported .mpp files back to Project Online or Project Server is not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NadinMerali, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Benzicald PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**